components/brave_wallet_ui/components/desktop/views/accounts/accounts.tsx

```diff
@@ -84,92 +80,117 @@ export const Accounts = () => {
+    <WalletPageWrapper
+      wrapContentInBox
+      cardHeader={
+        <AccountsHeader />
+      }
+    >
+      <Row
+        padding='8px'
+        justifyContent='flex-start'
+      >
+        <SectionTitle
+        >
+          {getLocale('braveWalletAccounts')}
+        </SectionTitle>
+      </Row>
+      <Column
+        fullWidth={true}
+        alignItems='flex-start'
+        margin='0px 0px 24px 0px'
+      >
+        {derivedAccounts.map((account) =>
+          <AccountListItem
+            key={account.accountId.uniqueKey}
+            onClick={onSelectAccount}
+            account={account}
           />
-        </ButtonRow>
-
-        <Column fullWidth={true} alignItems='flex-start'>
-          <SectionTitle>
-            {getLocale('braveWalletAccountsSecondary')}
-          </SectionTitle>
-          <DisclaimerText>
-            {getLocale('braveWalletAccountsSecondaryDisclaimer')}
-          </DisclaimerText>
-          <SubDivider />
-        </Column>
-
-        <SecondaryListContainer isHardwareWallet={false}>
-          {secondaryAccounts.map((account) =>
-            <AccountListItem
-              key={account.accountId.uniqueKey}
-              onClick={onSelectAccount}
-              account={account}
-            />
-          )}
-        </SecondaryListContainer>
-
-        {Object.keys(trezorAccounts).map(key =>
-          <SecondaryListContainer key={key} isHardwareWallet={true}>
-            {sortAccountsByName(trezorAccounts[key])
-              .map((account: WalletAccountType) =>
-                <AccountListItem
-                  key={account.accountId.uniqueKey}
-                  onClick={onSelectAccount}
-                  account={account}
-                />
-              )}
-          </SecondaryListContainer>
-        )}
-
-        {Object.keys(ledgerAccounts).map(key =>
-          <SecondaryListContainer key={key} isHardwareWallet={true}>
-            {sortAccountsByName(ledgerAccounts[key])
-              .map((account: WalletAccountType) =>
-                <AccountListItem
-                  key={account.accountId.uniqueKey}
-                  onClick={onSelectAccount}
-                  account={account}
-                />
-              )}
-          </SecondaryListContainer>
         )}
-
-        <ButtonRow>
-          <StyledButton onClick={onClickAddAccount('import')}>
-            <WalletIcon />
-            <ButtonText>{getLocale('braveWalletAddAccountImport')}</ButtonText>
-          </StyledButton>
-          <StyledButton onClick={onClickAddAccount('hardware')}>
-            <HardwareIcon />
-            <ButtonText>
-              {getLocale('braveWalletAddAccountImportHardware')}
-            </ButtonText>
-          </StyledButton>
-        </ButtonRow>
-    </>
+      </Column>
+
+      {importedAccounts.length !== 0 &&
+        <>
+          <Row
+            padding='8px'
+            justifyContent='flex-start'
+          >
+            <SectionTitle
+            >
```

Comment:
nit: closing bracket
```suggestion
            <SectionTitle>
```

Reply:
Fixed!